src/main.cpp

```diff
@@ -4512,6 +4518,15 @@ void static ProcessGetData(CNode* pfrom, const Consensus::Params& consensusParam
                         // else
                             // no response
                     }
+                    else if (inv.type == MSG_CMPCT_BLOCK)
+                    {
+                        // Note that while we will respond to all MSG_CMPCT_BLOCK requests here,
```

Comment:
In hindsight the DoS score is probably not a good idea here...gonna swap it so that we just dont respond, as well as remove the DoS score in the GETBLOCKTXN request.